spread.yaml

```diff
@@ -68,22 +73,32 @@ prepare: |
     sysctl -w net.ipv6.conf.all.disable_ipv6=1
     trap "sysctl -w net.ipv6.conf.all.disable_ipv6=0" EXIT
 
-    apt purge -y snapd || true
-    apt update
+    if [ $(lsb_release -c -s) = trusty ]; then
```

Comment:
Can you use /etc/os-release for this instead. I'd rather not add lsb-release here.